engine/src/script.cpp

```diff
@@ -140,7 +143,11 @@ bool Script::setData(const QString& str)
     if (m_data.isEmpty() == false)
     {
         int i = 1;
+#if QT_VERSION < QT_VERSION_CHECK( 5, 14, 0 )
         QStringList lines = m_data.split(QRegExp("(\r\n|\n\r|\r|\n)"), QString::KeepEmptyParts);
```

Comment:
So this last commit got pretty big, pretty boring monkey work (look up the matching Qt enum constants and use these instead of plain numbers (mostly 0). This was not possible in two occasions, as the corresponding enum  [Qt::Orientations](https://doc.qt.io/qt-5/qt.html#Orientation-enum) doesn't contain a value 0. I haven't found out how to inline the matching code from my commit here, so I name the files/line numbers:

ui/src/flowlayout.cpp:117
ui/src/monitor/monitorlayout.cpp:120

Both methods don't seem to be used anywhere in the code currently. So I set the return value to Qt::Vertical for now without understanding what the functions are intended to do - please correct me if this is wrong.

Reply:
I'm sorry, there are still missing bits as I see now via travis and appveyor failing, there need to be more version checks for using Qt::endl
I will keep committing stuff until the tests succeed...

Reply:
So travis still fails, with this:
FAIL!  : QLCInputProfile_Test::save() 'ip.saveXML(path) == false' returned FALSE. ()

   Loc: [qlcinputprofile_test.cpp(390)]

I've tried to alter this call locally but can't seem to get it right.
I have no clue which permissions to put there for the test to pass, and ORing more permission bits leads to ther more obscure errors here...

This is a bit beyond my understanding I have to admit.

Reply:
Thanks, i should have been able to try that myself...

Looks good now.